geonode/maps/templates/maps/map_detail.html

```diff
@@ -105,6 +105,7 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Download Map" %}</h4>
                 <ul>
                   <li><a href="{% url "map_download" resource.pk %}">{% trans "Download Data Layers" %}</a></li>
                   <li><a href="{% url "map_wmc" resource.pk %}" target="_new">{% trans "Download Web Map Context" %}</a></li>
+                  <li><a href="{% url "map_download_qlr" resource.pk %}" target="_new">{% trans "Download QLR Layer file" %}</a></li>
```

Comment:
@boney-bun Correct me if I am wrong, but this line will be displayed for QGIS and Geoserver. At least, we shouldn't display it for geoserver. Better to use links I guess like in layers.